src/Microsoft.SqlTools.ServiceLayer/Connection/Contracts/ConnectionDetails.cs

```diff
@@ -5,7 +5,7 @@
 
 using System;
 using System.Collections.Generic;
-using System.Globalization;
+using System.Globalization;
```

Comment:
This is unrelated to my change except that my change restored a unix line ending on this line.

Globalization is used near the bottom of the file by some code that isn't part of my change.